x/crisis/module.go

```diff
@@ -35,10 +36,7 @@ import (
 // ConsensusVersion defines the current x/crisis module consensus version.
 const ConsensusVersion = 2
 
-var (
-	_ module.EndBlockAppModule = AppModule{}
-	_ module.AppModuleBasic    = AppModuleBasic{}
-)
+var _ module.AppModuleBasic = AppModuleBasic{}
```

Comment:
We should verify here that the modules implements `appmodule.HasEndBlocker` instead of `module.EndBlockAppModule`.